packages/carbon-graphs/dev/examples/controls/bar.js

```diff
@@ -551,6 +551,55 @@ export const renderBarPanningWithDynamicData = (id) => {
   });
   return graph;
 };
+export const renderBarPanningWithDynamicEventline = (id) => {
+  const axisData = utils.deepClone(getDemoData(`#${id}`, "BAR_TIMESERIES"));
+  axisData.pan = {
+    enabled: true
+  };
+  axisData.eventline = [
+    {
+      color: Carbon.helpers.COLORS.GREY,
+      style: {
+        strokeDashArray: "4,4"
+      },
+      value: new Date(2016, 0, 1, 4, 30).toISOString()
+    }
+  ];
+  axisData.axis.x.lowerLimit = new Date(2016, 0, 1, 0).toISOString();
+  axisData.axis.x.upperLimit = new Date(2016, 0, 2, 0).toISOString();
+  axisData.axis.x.ticks = {
+    values: [
+      new Date(2016, 0, 1, 3).toISOString(),
+      new Date(2016, 0, 1, 6).toISOString(),
+      new Date(2016, 0, 1, 9).toISOString(),
+      new Date(2016, 0, 1, 12).toISOString(),
+      new Date(2016, 0, 1, 15).toISOString()
+    ],
+    format: "%H"
+  };
+  const createGraph = () => {
+    graphData[0].eventline = [
+      {
+        color: Carbon.helpers.COLORS.BLACK,
+        style: {
+          strokeDashArray: "2,2"
+        },
+        value: new Date(2016, 0, 1, 7, 30).toISOString()
+      }
+    ]
+    graph.reflow(graphData[0]);
```

Comment:
Did you tested by changing it to different reflow data say(graphData[1]). Because it is not working as expected from my side. please correct me if I am wrong.

Reply:
So in this test, I only wanted to show the change in eventline rather than updating the bars as well. But still, I have updated it with graphData[1] here.

Reply:
did you try the same thing in different graphs?

Reply:
Yes